src/components/header/header.js

```diff
@@ -0,0 +1,298 @@
+
+export default class HeaderComponent extends HTMLElement {
+    constructor() {
+        super();
+
+        const bodyElement = document.body;
+        bodyElement.style.margin = '0';
+        this.attachShadow({ mode: 'open' });
+        this.shadowRoot.innerHTML = `
+        
+        <header>
+            <div class="logo-bar">
+                <img class="greenwood-logo" src="/assets/greenwood-logo.svg" alt="Logo">
```

Comment:
we'll need an actual image file (e.g. _.png_, _.webp_, etc) here if we want to use an actual `<img>` tag.  Let me try and pull that over from the current site.